src/main/java/com/onarandombox/MultiverseNetherPortals/MultiverseNetherPortals.java

```diff
@@ -152,6 +146,24 @@ public void loadConfig() {
         this.saveMVNPConfig();
     }
 
+    private void initMVNPConfig() {
+        this.MVNPConfiguration = new YamlConfiguration();
+        try {
+            File configFile = new File(this.getDataFolder(), NETHER_PORTALS_CONFIG);
+            if (!configFile.isFile()) {
+                Logging.info("Creating new %s", NETHER_PORTALS_CONFIG);
```

Comment:
Do other plugins log when they create a new config file? I was thinking it could be `fine` instead of `info`. But if other plugins do the same thing then it's good.

Reply:
I think its fine, bcu should only happen ones anyways. And if they deleted the config, it will be good to tell them a new one is created.

Reply:
Okay, sure. Good idea 👍.